### PR TITLE
Upgrade to PSPDFKit 5.0.1 for Android

### LINKED ISSUE #35: Update to PSPDFKit 5 for Android
With yesterday's release of PSPDFKit 5 for Android we can now update the Cordova/Ionic wrapper.

# Details

* Update PSPDFKit used by the wrapper to 5.0.0.
* Check whether we can also bump Cordova Android used by this wrapper.

# Acceptance Criteria

* [ ] Update PSPDFKit for Android to version 5.0.0.

### PR DESCRIPTION
**Pull request by davidschreiber**
## Resolves #35 

This PR upgrades the Cordova wrapper to PSPDFKit 5.0.1 for Android. 

Right now, the latest stable version [7.1.4 of `cordova-android`](https://www.npmjs.com/package/cordova-android/v/7.1.4) still uses the outdated Android Gradle plugin 3.0.1, which is currently incompatible with this version of Cordova-Android. Since Cordova does not support forcing nightly builds using the `package.json` file, customers have to manually upgrade to the nightly version from the command line (version `8.0.0-nightly.2018.11.23.ef243418` was tested and works perfectly fine). I documented to use this version in the basic setup steps of our README. However, if customers run into upgrade issues, we might need to put together a manual migration guide too (I'd leave that open for now).

### Changes in this PR

* Bump versions of the wrapper to 5.0.1
* Simplify integration of PSPDFKit by switching to automated Maven integration instead of manual AAR file integration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,63 +40,63 @@ function showMyDocument() {
 Opens a document from the local device storage.
 
 ```javascript
-PSPDFKit.showDocument(uri, options, success, fail)
+PSPDFKit.showDocument(uri, options, success, fail);
```
Comment (anastasia-zaitsewa) [inline README.md]: ";" is optional in Javascript

File: README.md [hunk 2]
```diff
@@ -42,63 +40,63 @@ function showMyDocument() {
...
 ```
 
 ## PSPDFKit.showDocumentFromAssets
 
 Opens a document from the app's asset directory. To package a file within your app's assets, put it into the `www/` directory of your project.
 
 ```javascript
-PSPDFKit.showDocumentFromAssets(assetPath, options, success, fail)
+PSPDFKit.showDocumentFromAssets(assetPath, options, success, fail);
```
Comment (anastasia-zaitsewa) [inline README.md]: ";" is optional in Javascript

**Comment by davidschreiber** [comment]
The blocking PR was merged, but the changes haven't been released in a stable build yet. Currently, it seems as if the published `8.0.0-nightly.2018.11.23.ef243418` nightly of `cordova-android` should work with PSPDFKit 5.0.1: https://www.npmjs.com/package/cordova-android/v/8.0.0-nightly.2018.11.23.ef243418

**Comment by davidschreiber** [comment]
Tested and verified that we're running against the current nightly `8.0.0-nightly.2018.11.23.ef243418`. We will evaluate if it is sensible to document how to use the nightly, or whether we should wait for the next stable release version of cordova-android.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pspdfkit-cordova-android",
-  "version": "4.3.0",
+  "version": "5.0.14.3.0",
```
[the remark this answers is not in the document]
Comment (davidschreiber) [reply package.json]: Thanks!

**Pull request merged**